src/pudl/metadata/resources/ferc1.py

```diff
@@ -1737,7 +1746,24 @@
         "field_namespace": "ferc1",
     },
     "out_ferc1__yearly_all_plants": {
-        "description": "Denormalized table combining the steam, small generators, hydro, and pumped storage tables from FERC Form 1.",
+        "description": {
+            "additional_summary_text": "plants reported in the steam, small generators, hydro, and pumped storage tables",
```

Comment:
Sneaky missing period
```suggestion
            "additional_summary_text": "plants reported in the steam, small generators, hydro, and pumped storage tables.",
```